### ISSUE TITLE
No module named 'SimpleITK'

### ISSUE THREAD

**Issue by zoujinger** [SuperElastix/SimpleElastix#135]
Hello, everyone

I build SimpleElastix  on my PC(Windows 10 64bit), with CMake and visual studio 2015. 
However when I try to import SimpleITK into IPython3.6 ,I  get the following error:
>>> import SimpleITK as sitk
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'SimpleITK'

But I can import SimpleElastix  with following command: 
>>> import SimpleElastix
>>>

Any help is very much appreciated and thanks in advance.

**Comment by kaspermarstal** [comment]
Are you using the latest version? A while ago I accidentally committed the wrong name for the Python version, but this was reverted recently.

**Comment by zoujinger** [comment]
Yes, I am using the latest version, download from github. I re-installed  it and I still cannot import SimpleITK  with Python 3.6

**Comment by zoujinger** [comment]
I checked the source file and find that the date modified was 9/27/2017 6:30 AM

**Comment by zoujinger** [comment]
I build with command line, and following is the error:
"C:\Users\studentx\build\ALL_BUILD.vcxproj" (default target) (1) ->
"C:\Users\studentx\build\SimpleITK.vcxproj" (default target) (11) ->
(CustomBuild target) ->
  python36.lib(python36.dll) : fatal error LNK1112: module machine type 'x64' conflicts with target machine type 'X86'
[C:\Users\studentx\build\SimpleITK-build\Wrapping\Python\SimpleITK_PYTHON.vcxproj] [C:\Users\studentx\build\SimpleITK.v
cxproj]
  python36.lib(python36.dll) : fatal error LNK1112: module machine type 'x64' conflicts with target machine type 'X86'
[C:\Users\studentx\build\SimpleITK-build\Wrapping\Python\SimpleITK_PYTHON.vcxproj] [C:\Users\studentx\build\SimpleITK.v
cxproj]

    26 Warning(s)
    2 Error(s)

**Comment by kaspermarstal** [comment]
Make sure the python libs are the same bitness (x86 or x64) as the compiler and linker. That is all I can say to help you from the error message.

**Comment by TrinhDinhPhuc** [comment]
:( i am fucked up with this issue

**Comment by kaspermarstal** [comment]
Feel free to reopen if this is still an issue after the update to SimpleITK 1.1.
